src/test/java/seedu/address/model/AddressBookTest.java

```diff
@@ -83,6 +83,14 @@ public void getPersonList_modifyList_throwsUnsupportedOperationException() {
         assertThrows(UnsupportedOperationException.class, () -> addressBook.getPersonList().remove(0));
     }
 
+    @Test
+    public void toStringMethod() {
+        String expected = AddressBook.class.getCanonicalName() + "{persons=" + addressBook.getPersonList() + "}";
+        System.out.println(addressBook.toString());
```

Comment:
You're using System.out.println() to print the results of the toString() method and the expected output. This is likely helpful for your debugging process. However, it's best to remove these lines once your test is working correctly. Test methods ideally should not produce console output, as it can clutter the test result reports.

Reply:
Good catch @SPWwj